database_schemas/cassandra/benchflow.cql

```diff
@@ -233,6 +234,32 @@ CREATE TABLE exp_cpu (
   PRIMARY KEY (experiment_id, container_id)
 );
 
+CREATE TABLE exp_cpu_core (
+  cpu_weighted_avg double,
+  cpu_q1_min double,
+  cpu_q1_max double,
+  cpu_q2_min double,
+  cpu_q2_max double,
+  cpu_q3_min double,
+  cpu_q3_max double,
+  cpu_p95_min double,
+  cpu_p95_max double,
+  cpu_ci095_min double,
+  cpu_ci095_max double,
+  cpu_median_min double,
+  cpu_median_max double,
+  cpu_min double,
+  cpu_max double,
+  cpu_sd double,
+  cpu_best list<text>,
+  cpu_average list<text>,
+  cpu_worst list<text>,
+  experiment_id text,
+  container_id text,
+  core bigint,
```

Comment:
@VincenzoFerme Maybe I was thinking too high. I don;t know how many cores there could potentially be, so I did that.